ta4j-core/src/main/java/org/ta4j/core/analysis/criteria/NumberOfMaxPositionsCriterion.java

```diff
@@ -29,9 +29,9 @@
 import org.ta4j.core.num.Num;
 
 /**
- * Number of position criterion.
+ * Number of maxium allowed position criterion.
  */
-public class NumberOfPositionsCriterion extends AbstractAnalysisCriterion {
+public class NumberOfMaxPositionsCriterion extends AbstractAnalysisCriterion {
```

Comment:
Why renaming this?

Reply:
Currently the betterThan() with NumberOfPositionsCriterion is vague. Look at the current impl of `betterThan()`. **Why is having more positions better?** Thus, split this criterion to two and let the user decide if one is better than the other. For example, for backtesting:

    NumberOfMinPositionsCriterion: the more positions, the more tests were done, the better.
    NumberOfMaxPositionsCriterion: the less positions, the better.

That's the idea why I wanted to split it to two criterions. Should I add new criterions instead? `NumberOfMinMaxPositionsCriterion` and leave the current as it is?